binrz/man/rz-hash.1

```diff
@@ -32,53 +28,57 @@ This command can be used to calculate hashes of a certain part of a file or a co
 .Pp
 This is the command used by the 'ph' command of rizin.
 .Bl -tag -width Fl
+.It Fl v
+Shows version number
+.It Fl h
+Show usage help message
+.It Fl 
+Input read from stdin instead of from a file
 .It Fl a Ar algo
-Select an algorithm for the hashing. Valid values are listed in: rz-hash -L
-.It Fl b Ar blocksize
-Define the block size
-.It Fl c Ar hash
-Compare the computed hash with this one. Allowed only when a single hash is computed.
+Hash algorithm to use; you can specify multiple ones by appending a comma (example: sha1,md4,md5,sha256)
+.It Fl B
+Outputs the calculated value for each block
+.It Fl b Ar size
+Sets the block size
```

Comment:
Same, `Set` not `sets`, here in other places, `output` not `outputs`, etc, you got the idea.